### ISSUE TITLE
rgss3a decompilation when Data folder is not empty

### ISSUE THREAD

**Issue by Fair95** [AhmedAhmedEG/VX-Ace-Translator#4]
As the title suggested, I had games with rgss3a file while the Data folder is not empty. The author only puts translation files there but not rvdata2 files. In this case, the decompiler is not working.

**Comment by AhmedAhmedEG** [comment]
Well, that should not be that big of a deal, I may add a new command line arg and make the default Data folder name a variable, but I don't have the time currently, when I get the time I will work on it.

**Comment by Fair95** [comment]
> Well, that should not be that big of a deal, I may add a new command line arg and make the default Data folder name a variable, but I don't have the time currently, when I get the time I will work on it.

Yeah, I have it fixed myself by manually deleting Data folder, but just letting you be aware of that issue since people may not be so familiar with rpg file structure, data types and so on. At least, it takes me some time to find out why the decompiler fails to work. Anyway, thanks for the great tool. And I will close this issue for now.

**Comment by AhmedAhmedEG** [comment]
Well, that should not be that big of a deal, I may add a new command line arg and make the default Data folder name a variable, but I don't have the time currently, when I get the time I will work on it.

> > Well, that should not be that big of a deal, I may add a new command line arg and make the default Data folder name a variable, but I don't have the time currently, when I get the time I will work on it.
> 
> Yeah, I have it fixed myself by manually deleting Data folder, but just letting you be aware of that issue since people may not be so familiar with rpg file structure, data types and so on. At least, it takes me some time to find out why the decompiler fails to work. Anyway, thanks for the great tool. And I will close this issue for now.

Your wellcome! also no need to close the issue, leave it open so until it's done.
